Include/internal/pycore_global_objects_fini_generated.h

```diff
@@ -12,9 +12,11 @@ extern "C" {
 static inline void
 _PyStaticObject_CheckRefcnt(PyObject *obj) {
     if (Py_REFCNT(obj) < _Py_IMMORTAL_REFCNT) {
-        _PyObject_ASSERT_FAILED_MSG(obj,
-            "immortal object has less refcnt than expected "
-            "_Py_IMMORTAL_REFCNT");
+        PyObject *repr = PyObject_Repr(obj);
```

Comment:
You should use `_PyObject_Dump` not repr as earlier code was doing.

Reply:
Great point that looks much better